xarray/datatree_/datatree/ops.py

```diff
@@ -1,6 +1,6 @@
 import textwrap
 
-from xarray import Dataset
+from xarray.core.dataset import Dataset
```

Comment:
This was causing another circular dependency issue. @flamingbear - just FYI, for when you are tweaking `ops.py`.